index.html

```diff
@@ -83,6 +83,28 @@
     </div>
 </div>
 
+<div id="download-complete-modal" class="modal">
+    <div class="modal-content">
+        <h4>Thank you for downloading Impact!</h4>
+        <p id="mac-moment">
+            On Mac, you may run into the following issue with "Unidentified Developer" on opening the Installer. <a href="https://github.com/ImpactDevelopment/ImpactClient/wiki/Opening-the-installer-on-Mac">Click here</a> for instructions on how to get around this issue.<br />
+            <img src="img/macmoment.png" style="width: 50%">
+        </p>
+        <script type="text/javascript">
+            if (navigator.platform.toUpperCase().indexOf('MAC') == -1) { // if NOT on Mac OS X
+                var elem = document.getElementById("mac-moment"); // then delete the mac info
+
+                elem.parentNode.removeChild(elem);
+            }
```

Comment:
I love this, show the message _unless_ they have javascript and we are sure they aren't on mac

Reply:
yeah some dudes probably have noscript LOL